test/git_chain/command/prune_test.rb

```diff
@@ -3,7 +3,7 @@
 
 module GitChain
   module Commands
-    class RebaseTest < MiniTest::Test
+    class PruneTest < MiniTest::Test
```

Comment:
Thanks for catching that. Fixed on master, please rebase.

https://github.com/Shopify/git-chain/pull/26